README.md

```diff
@@ -54,7 +63,7 @@ To finish the general setup you can run these commands to build all containers:
 ```bash
 invoke prepare-builds
 aws ecr get-login-password --region eu-central-1 | docker login --username AWS --password-stdin 017973353230.dkr.ecr.eu-central-1.amazonaws.com
-docker-compose -f docker-compose.yml up --build
+docker-compose -f docker-compose.yml -f docker-compose.override.yml up --build | docker login --username AWS --password-stdin 017973353230.dkr.ecr.eu-central-1.amazonaws.com
```

Comment:
One logged in to docker you shouldn't have to re-login for every command as far as I know. Why is the command on line 65 not logging you in?